packages/dataviews/src/style.scss

```diff
@@ -285,9 +285,9 @@
 		width: 100%;
 		min-height: 200px;
 		aspect-ratio: 1/1;
-		overflow: hidden;
 		border-bottom: 1px solid $gray-200;
 		background-color: $gray-100;
+		border-radius: 3px 3px 0 0;
```

Comment:
Why is the rounded corner value 3px in this PR? How about making the corners the same as the rounded corners of the card itself?

```
border-radius: $radius-block-ui * 2 $radius-block-ui * 2 0 0;
```

Reply:
If it matches precisely then there's a small space where the background is visible between the card and the media wrapper / button. You have to zoom in to see it:

<img width="240" alt="Screenshot 2024-01-23 at 10 57 43" src="https://github.com/WordPress/gutenberg/assets/846565/ba508b04-ed9a-413a-b841-0c549ae315b7">

I appreciate we could avoid the additional `border-radius` styles if we apply `overflow: hidden` to the card, but then the button focus style would be hidden. 

If you have another idea how to tackle this please let me know, or feel free to push to the branch :)